mod/wikirate/set/right/general_overview.rb

```diff
@@ -0,0 +1,36 @@
+format :html do
+  view :content do |args|
+    handle_edit_general_overview( args ) { super args }
+  end
+
+  view :missing do |args|
+    handle_edit_general_overview( args ) { super args }
+  end
+
+  view :titled_with_edits do |args|
+    handle_edit_general_overview( args ) { super args }
+  end
+
+  view :editor do |args|
+    if claim_name = params[:citable] and claim = Card[claim_name]
+      prompt = with_inclusion_mode :normal do
+        nest claim, :view=>:sample_citation
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Surrounding space missing for operator `=>`.